plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java

```diff
@@ -165,8 +165,7 @@ public CortexTSS(final CortexTSSConfig config) {
         metrics.register("maxAllowedConcurrentCalls", (Gauge<Integer>) () -> asyncHttpCallsBulkhead.getMetrics().getMaxAllowedConcurrentCalls());
     }
 
-    @Override
-    public void store(final List<Sample> samples) throws StorageException {
+    public void store(final List<Sample> samples, String clientID) throws StorageException {
```

Comment:
Let's add a new method instead, and have the existing method call in the the value from the config.

We can take a similar approach with the other methods such as `findMetrics` and `getTimeseries`.

Reply:
I see, good idea. How those findMetrics and getTimeseries methods using the client ID? I didn't find information in the cortex API document.

Reply:
We need to set the value of the `X-Scope-OrgID` header for all the HTTP requests we make.